token_filter.module

```diff
@@ -11,8 +10,9 @@
 function token_filter_filter_info() {
   $filters['filter_tokens'] = array(
     'title' => t('Replace tokens'),
-    'description' => t('The usage of this filter should be restricted to trusted users only as tokens with sensitive data could be exposed.'),
+    'description' => t('The usage of this filter might need to be restricted to trusted users only, as tokens with sensitive data could be exposed. You can also limit the tokens replaced by this filter in its configuration.'),
     'process callback' => '_token_filter_filter_tokens',
+    'settings callback' => '_token_filter_settings',
     'tips callback' => '_token_filter_filter_tips',
```

Comment:
@bugfolder 
On the basis of @argiepiano 's great detective 🕵️ skills, here is a suggested change to the PR:

```suggestion
    'settings callback' => '_token_filter_settings',
    'default settings' => array(
      'replace' => 0,
      'tokens' => '',
    ),
    'tips callback' => '_token_filter_filter_tips',
```